### PR TITLE
OP-1721: Added family status dropdown

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
Ticket: https://openimis.atlassian.net/browse/OP-1721

BE: https://github.com/openimis/openimis-be-insuree_py/pull/113

Changes:
- Added new dropdown to filter by family status.
- Insurees without family are shown with different background on an insuree's list.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -30,3 +31,4 @@ export const DEFAULT = {
 };
 
 export const WITHOUT_STR = "without"
+export const FAMILY_WITHOUT_STR = "without"
```
Comment (olewandowski1) [inline src/constants.js:34]: Does creating 'FAMILY_WITHOUT_STR' make sense? 'WITHOUT_STR' seems quite general.

File: src/pickers/FamilyStatusPicker.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React, { Component } from "react";
```
Comment (olewandowski1) [inline src/pickers/FamilyStatusPicker.js:1]: 'Component' is not used.

File: src/constants.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 export const INSUREE_MARITAL_STATUS = ["N", "W", "S", "D", "M"];
 export const FAMILY_POVERTY_STATUS = [true, false];
 export const PHOTO_STATUS = ["with", "without"];
+export const FAMILY_STATUS = ["with", "without"];
```
Comment (olewandowski1) [inline src/constants.js:4]: These variables can be combined into a single one with a generic name.

**Review by olewandowski1 — CHANGES_REQUESTED** [review]
I left some comments. @wzglinieckisoldevelo

File: src/constants.js (hunk 1 above)
Comment (wzglinieckisoldevelo) [reply src/constants.js:34]: On one hand yes, but I wanted to separate them - if for some reason WITHOUT_STR has to be changed for other feature they would have to be separated anyway.

File: src/constants.js (hunk 3 above)
Comment (wzglinieckisoldevelo) [reply src/constants.js:4]: Same as above - if for example we have another status for photo (let's say with/without/expired) we would have to reintroduce this variable for family_status.

Comment (olewandowski1) [reply src/constants.js:4]: Makes sense, up to you.

File: src/constants.js (hunk 1 above)
Comment (olewandowski1) [reply src/constants.js:34]: I don't think WITHOUT_STR will change. If it does, it won't be WITHOUT_STR anymore.

**Review by olewandowski1 — APPROVED** [review]
LGTM

**Pull request merged**